### PR TITLE
Task Feature : General UI-UX Improvements

### PR DESCRIPTION
**Pull request by kumarpalsinh25**
Ref #1841,

1) My Task : Home Screen
- Now we have compact view of my task item list
- Task List have better representation by leading icon

Screenshot 2024-06-22 at 12 24 48 AM

2) Task Item Detail Page:

https://github.com/acterglobal/a3/assets/51526480/362841e4-5ef8-4c93-81c6-c4d32ad8e902

### CODE REVIEW

File: app/lib/features/tasks/pages/task_item_detail_page.dart [hunk 1]
```diff
@@ -353,11 +356,7 @@ class TaskItemDetailPage extends ConsumerWidget {
     return ListTile(
       dense: true,
       leading: const Icon(Atlas.business_man_thin),
-      title: Text(
-        L10n.of(context).assignment,
-        style: Theme.of(context).textTheme.bodyMedium,
-      ),
-      subtitle: task.isAssignedToMe()
+      title: task.isAssignedToMe()
```
Comment (gnunicorn) [inline app/lib/features/tasks/pages/task_item_detail_page.dart:359]: Something I might have overlooked before but whey do we not show the assigned if it isn't the user themselves? This check seems wrong...

**Review by gnunicorn — APPROVED** [review]
Lgtm but one thing looks like a bug that was already in even before

File: app/lib/features/tasks/pages/task_item_detail_page.dart (hunk 1 above)
Comment (kumarpalsinh25) [reply app/lib/features/tasks/pages/task_item_detail_page.dart]: Ahh I see,
I have did required changes in the task details page as below.

Screenshot 2024-06-25 at 10 09 49 AM

Comment (kumarpalsinh25) [reply app/lib/features/tasks/pages/task_item_detail_page.dart]: How ever I find some additional issue on assignment which I reported here.
https://github.com/acterglobal/a3/issues/1877

**Pull request merged**